backend/apps/issue_quality/views.py

```
@@ -0,0 +1,104 @@
+
+    @action(detail=False, methods=['get'])
+    def stats(self, request):
+        """
+        Get quality statistics.
+        """
+        total = IssueQualityCheck.objects.count()
+        if total == 0:
+            return Response({'message': 'No data available'})
+        
+        avg_quality = IssueQualityCheck.objects.aggregate(
+            avg=models.Avg('quality_score')
```

Ruff (0.15.21)

[error] Undefined name `models`

(F821)